uint8 num
float32 radius
int32 angleDegrees
float32 interCoeff
float32 propCoeff
float32 diffCoeff
float32 coeff
float32 turnMaxSpeed
float32 maxSpeed 
int32 ticksPerRotation